TACC/Core-CMS-Custom
# Task/wp 425 admin user listing

## Overview
convert administration/view-users/ table, "View Users" to React

## Related
- [WP-425](https://tacc-main.atlassian.net/browse/WP-425)
- requires https://github.com/TACC/Core-CMS/pull/117

## Changes


## Testing

1. pull down branch
2. in .../src/client, run `npm run build`
3. Compare table to original
4. Verify the Status and Organization filters work as intended

## UI
New:
![Screen Shot 2024-06-11 at 4 08 50 PM](https://github.com/TACC/Core-CMS-Custom/assets/35277477/d5a60394-e1b7-4e20-8b08-600819fb10f1)

Original:
![Screen Shot 2024-06-11 at 10 27 41 AM](https://github.com/TACC/Core-CMS-Custom/assets/35277477/375d5262-aba5-48c9-8965-1af171abce75)


## Notes
'See More' dropdown opens modals. Those modals do not exist (?), so choosing an option does nothing.
